planning/behavior_velocity_planner/src/utilization/path_utilization.cpp

```diff
@@ -193,11 +205,14 @@ autoware_auto_planning_msgs::msg::Path interpolatePath(
   const auto y_interp = interpolation::slerp(s_in, y, s_out);
   const auto z_interp = interpolation::slerp(s_in, z, s_out);
 
+  // Find a nearest segment for each point in s_out and use the velocity of the segment's beginning
+  // point. Note that if s_out is almost the same value as s_in within DOUBLE_EPSILON range, the
+  // velocity of s_out should be same as the one of s_in.
   std::vector<double> v_interp;
   size_t closest_segment_idx = 0;
   for (size_t i = 0; i < s_out.size() - 1; ++i) {
     for (size_t j = closest_segment_idx; j < s_in.size() - 1; ++j) {
-      if (s_in.at(j) < s_out.at(i) + DOUBLE_EPSILON && s_out.at(i) < s_in.at(j + 1)) {
+      if (s_in.at(j) - DOUBLE_EPSILON < s_out.at(i) && s_out.at(i) < s_in.at(j + 1)) {
```

Comment:
@TakaHoribe 
is this condition different from previous?

Reply:
Just make it easy to read.